tests/System.Drawing.Tests/ColorTests.cs

```diff
@@ -57,5 +57,109 @@ public void ColorFromKnownColorTest()
             Assert.Equal((byte)0x00, color.G, "G should be 0x00");
             Assert.Equal((byte)0x00, color.B, "B should be 0x00");
         }
+
+        // Method set to private
```

Comment:
Please remove dead code or explicitly write why you'd like to keep this dead code. In this case, not sure why ;-)